andela/travel_tool_back
# Add the relevant occupation names

#### What does this PR do?
This PR properly formats and adds a few demoable occupations for the demo.

#### Description of Task to be completed?
This PR is due to the fact that a lot of occupation name relevant to the Demo are missing and this PR adds them.

#### How should this be manually tested?
- Clone the repo
- Run `make start`
- Run `make ssh`
- Run `yarn db:seed`
- Try creating a new request you should see the occupations in capital letter and a few more added.

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
N/A

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
